npm/pod.go

```diff
@@ -215,36 +215,42 @@
 		npMgr.NsMap[podNs], err = newNs(podNs)
 		if err != nil {
 			metrics.SendErrorLogAndMetric(util.PodID, "[AddPod] Error: failed to create namespace %s with err: %v", podNs, err)
+			return err
```

Comment:
>return err [](start = 3, length = 10)

I think we need a workerQueue model before we take this to production. Currently for intermittent errors, we can miss events during resync as well. With workQueue controller, we will not mark the entry done until the call succeeds.